### ISSUE TITLE
Emergency Shutdown, SUCCESSFUL result.

### ISSUE THREAD

**Issue by whoissatoshi** [AcalaNetwork/acala-dapp#241]
1. Login
2. **Self Serviced Loan** tab
3. **Emergency Shutdown is Triggered**
4. Click **Next** button -->  Click **Next** button --> Click **Reclaim** button.
5. Confirm reclaim data and sign tx.

Actual Result:  YOUR RECLAIM WAS SUCCESSFUL! but tx was not made due to low balance.
Expected Result:  Warning about low balance.

5F2FNhhVeG9bW4tP5te4a7asBJvSHQg31RzkoETLpUN161gV

![Selection_040](https://user-images.githubusercontent.com/69153826/90253145-5d72b980-de49-11ea-87fd-dd081b3e1ffd.png)

**Comment by jiangfuyao** [comment]
@whoissatoshi `ACA` is required as a handling fee.  Need swap some ACA.

**Comment by whoissatoshi** [comment]
> @whoissatoshi `ACA` is required as a fee. Need swap some ACA.

Correct, I should have ACA. But the defect is about SUCCESS message. I see it, when tx failed

**Comment by qwer951123** [comment]
tracked at #247
